executor/main_test.go

```diff
@@ -44,6 +44,8 @@ func TestMain(m *testing.M) {
 		goleak.IgnoreTopFunction("go.etcd.io/etcd/pkg/logutil.(*MergeLogger).outputLoop"),
 		goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"),
 		goleak.IgnoreTopFunction("gopkg.in/natefinch/lumberjack%2ev2.(*Logger).millRun"),
+		goleak.IgnoreTopFunction("github.com/pingcap/tidb/table/tables.mockRemoteService"),
+		goleak.IgnoreTopFunction("github.com/pingcap/tidb/table/tables.NewCachedTable"),
```

Comment:
Why ignore this one?

Reply:
NewCachedTable called mockRemoteService . Maybe I just ignore it. It should be okay

Reply:
So please remove it